.expeditor/habitat_test.yml

```diff
@@ -0,0 +1,21 @@
+---
+steps:
+
+- label: ":linux: Validate Habitat Builds of Chef InSpec"
+  commands:
+    - /workdir/.expeditor/buildkite/habitat_test.sh
+  expeditor:
+    executor:
+      linux:
+        privileged: true
+        single-use: true
+
+- label: ":windows: Validate Habitat Builds of Chef InSpec"
+  commands:
+    - /workdir/.expeditor/buildkite/habitat_test.ps1
+  expeditor:
+    executor:
+      windows:
+        os-version: 2016
```

Comment:
should be able to remove `os-version: 2016` and use the default Windows 2019

Reply:
Ok thank you =)